### PR TITLE
[DOC] Update migrate static for tracing

### PR DESCRIPTION
**Pull request by knylander-grafana**
#### PR Description

Adds traces to the migration from static to Alloy documentation. 

Additional changes: 
* Change name of Grafana Cloud Tempo to Grafana Cloud Traces, since this is the product name 
* Change grafana_cloud_tempo to grafana_cloud_tempo

#### Which issue(s) this PR fixes

Part of https://github.com/grafana/tempo/issues/3648

#### PR Checklist

- [ ] CHANGELOG.md updated
- [X] Documentation added
- [ ] Tests updated
- [ ] Config converters updated

### CODE REVIEW

File: docs/sources/tasks/migrate/from-static.md [hunk 1]
```diff
@@ -197,12 +217,12 @@ prometheus.scrape "metrics_test_local_agent" {
...
     endpoint {
-        name = "test-3a2a1b"
+        name = "test-4dec64"
         url  = "https://prometheus-us-central1.grafana.net/api/prom/push"
 
         basic_auth {
-            username = "<USERNAME>"
-            password = "<PASSWORD>"
+            username = "USERNAME"
+            password = "PASSWORD"
```
Comment (clayton-cornell) [inline docs/sources/tasks/migrate/from-static.md:225]:
```suggestion
            username = "<USERNAME>"
            password = "<PASSWORD>"
```

We should keep the < and > for variables... and define them under the code block if necessary as well.

File: docs/sources/tasks/migrate/from-static.md [hunk 2]
```diff
@@ -169,6 +172,23 @@ logs:
...
+            grpc:
+            http:
+      batch:
+        send_batch_size: 10000
+        timeout: 20s
+      remote_write:
+        - endpoint: tempo-us-central1.grafana.net:443
+          basic_auth:
+            username: USERNAME
+            password: PASSWORD
```
Comment (clayton-cornell) [inline docs/sources/tasks/migrate/from-static.md:191]:
```suggestion
            username: "<USERNAME>"
            password: "<PASSWORD>"
```

I'm guessing these need double quotes too?

Comment (knylander-grafana) [reply docs/sources/tasks/migrate/from-static.md:191]: Honestly, I'm not sure. I'll defer to your judgement on this. @hedss Do you know?

Comment (knylander-grafana) [reply docs/sources/tasks/migrate/from-static.md:191]: If what I found in the Static mode doc is correct, I don't think we need the double quotes. 
```suggestion
            username: <USERNAME>
            password: <PASSWORD>
```

**Review by clayton-cornell — APPROVED** [review]
Looks OK to me. No more comments beyond what we've already discussed.

**Comment by knylander-grafana** [comment]
@clayton-cornell  I think we're good with this. Do you want to merge it?

**Pull request merged**